### DISCUSSION TITLE
Unsupported engine in latest renovate/renovate docker image

### DISCUSSION THREAD

**Discussion by joelgrimberg** [renovatebot/renovate#14530] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

@latest

### Please select which platform you are using if self-hosting.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

Linux

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

somehow my renovate does handle the node-version of my project very well.
I use the latest renovate/renovate docker image, but when I run node -v / npm -v , I get respectively 
$ npm -v
6.14.16
$ node -v
v14.19.0

is there a way to bump the node / npm versions of the Renovate Runner / Bot ?

```
      "prTitle": "fix(deps): update dependencies (non-major)"
 WARN: artifactErrors (repository=proj/frontend, branch=renovate/dependencies-(non-major))
       "artifactErrors": [
         {
           "lockFile": "package-lock.json",
           "stderr": "npm ERR! code ENOTSUP\nnpm ERR! notsup Unsupported engine for proj@1.47.3: wanted: {\"node\":\"^16.5.0\",\"npm\":\">=7\"} (current: {\"node\":\"14.19.0\",\"npm\":\"6.14.16\"})\nnpm ERR! notsup Not compatible with your version of node/npm: proj@1.47.3\nnpm ERR! notsup Not compatible with your version of node/npm: proj@1.47.3\nnpm ERR! notsup Required: {\"node\":\"^16.5.0\",\"npm\":\">=7\"}\nnpm ERR! notsup Actual:   {\"npm\":\"6.14.16\",\"node\":\"14.19.0\"}\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /tmp/renovate/cache/others/npm/_logs/2022-03-04T15_08_15_364Z-debug.log\n"
         }
       ]
```

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment, marked as answer]
If you need custom node versions today, you need to build your own Renovate image. Here's the Dockerfile it's based on: https://github.com/renovatebot/docker-renovate-full/blob/main/Dockerfile

We will shortly default to node 16/npm 8 though in our prebuilt image, you can track that issue here: https://github.com/renovatebot/docker-renovate-full/issues/406
